org.eclipse.jdt.core.compiler.batch/src/org/eclipse/jdt/internal/compiler/ast/GuardedPattern.java

```diff
@@ -209,6 +209,10 @@ public void wrapupGeneration(CodeStream codeStream) {
 		this.primaryPattern.wrapupGeneration(codeStream);
 	}
 	@Override
+	public void fullWrapupGeneration(CodeStream codeStream) {
+		this.primaryPattern.fullWrapupGeneration(codeStream);
+	}
+	@Override
 	protected void generatePatternVariable(BlockScope currentScope, CodeStream codeStream, BranchLabel trueLabel,
```

Comment:
Much water has flown under the bridge - the method wrapupGeneration doesn't exist anymore on master and it doesn't likely make sense to introduce this.

Reply:
I've rebased this PR against your changes in https://github.com/eclipse-jdtls/eclipse-jdt-core-incubator/pull/9 , I'll just need to replicate those changes here